modules/ROOT/pages/anypoint-custom-metrics-connector.adoc

```diff
@@ -1,47 +1,105 @@
-= Anypoint Custom Metrics Connector
+= Custom Metrics
```

Comment:
Why remove "connector"? Would I believe I had found the right topic if my search was "custom metrics connector"?

Reply:
It's complicated... :-) But if you search, you should end up on the Custom Metrics Connector doc in the connectors documentation.